web/src/main/java/com/microsoft/applicationinsights/web/internal/httputils/AIHttpServletListener.java

```diff
@@ -70,6 +72,9 @@ public void onError(AsyncEvent event) throws IOException {
                 if (throwable instanceof Exception) {
                     // AI SDK can only track Exceptions. It doesn't support tracking Throwable
                     handler.handleException((Exception) throwable);
+                } else {
```

Comment:
curious - why do we want to log this? If we don't support Throwable, should we just keep silent?

Reply:
I think it's good to have this log message, because when users create incidents against why exception is not getting captured, we can ask them to turn on logs and understand if their app is actually throwing a throwable. Good for diagnosis in my opinion.